### DISCUSSION TITLE
Allow text fields to be percieved as numeric on the form

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5700] [Feature Requests]
_I think I may have opened a request for this feature, so if found, please go ahead and close this issue._

In this instance, a collection wants to use a text field in the Collecting Event table as a numeric field. They want to track the Sampling Effort Value, but it must be a numeric field with decimals. There are no such fields in that table but we have extra text fields. It would be nice to have the form validate the text fields as if they are numeric if specificed in the form definition.

**Requested By:** Montreal Insectarium & NMSU Herbarium

**Comment by maxpatiiuk** [comment]
This feature was removed in 7.7 because I didn't consider this use case
Need to bring it back:
If user explicitly said in the form definition that field should be rendered as a number, then render it as a number even if it is a text field in the database

This is useful for people who run out of number fields but have spare text fields
